### PR TITLE
[Okex] rename into OKX

### PR DESCRIPTION
**Pull request by GuillaumeDSM**

### CODE REVIEW

File: dev_requirements.txt [hunk 1]
```diff
@@ -17,3 +17,5 @@ wheel
 pur
 
 pylint
+
+ccxt==1.79.91   # use to run test using a specific version of ccxt
```
Comment (Herklos) [inline dev_requirements.txt:21]: I think we should prefer `ccxt>=1.79.91`

Comment (GuillaumeDSM) [reply dev_requirements.txt:21]: Tests are not passing if we do this due to changes in bybit (and we are currently using this exact version on octobot)

Comment (Herklos) [reply dev_requirements.txt:21]: Ok, I think we'll have to do this to prevent incompatibility in the future

Comment (GuillaumeDSM) [reply dev_requirements.txt:21]: yep, especially at the moment when we can't update ccxt

**Pull request merged**